### DISCUSSION TITLE
Update the link to the X (twitter)

### DISCUSSION THREAD

**Discussion by 0xsingular** [teaxyz/white-paper#111] [General]
In the email received from tea about the latest news and updates in the protocol there is a link to the old twitter page, it would be great to have it updated.
![IMG_3426](https://github.com/teaxyz/white-paper/assets/35407208/aff65e75-63d2-4ffb-96c3-ec2915e9aea4)

**Comment by thomas-borrel** [comment]
Thank you @0xsingular - We'll let the team know
